personalize.admin.inc

```diff
@@ -388,10 +389,8 @@ function personalize_agent_list() {
         array('data' => $agent->plugin),
         array('data' => drupal_render($status_change_form)),
         //array('data' => $storage),
-        array('data' => l(t('Edit'), 'admin/structure/personalize/manage/'. $agent->machine_name)),
+        array('data' => $allow_edit ? l(t('Edit'), 'admin/structure/personalize/manage/'. $agent->machine_name . '/edit') : ''),
```

Comment:
I have to fix a bunch of these in my clean-up ticket, but we don't need the "edit" at the end.  The default state is just the agent name and then you can append one of subform steps to go directly to it.